style_guide.md

```diff
@@ -334,20 +334,20 @@ A string SHOULD be enclosed in single quotes if it does not contain variable int
         ```
 
 
-If a string is a value from an enumerable set of options, such as `present` and `absent`, it SHOULD NOT be enclosed in quotes at all.
+If a string is a value from an enumerable set of options, such as `present` and `absent`, it SHOULD be enclosed in quotes just like any other string.
 
 For example:
 
 Good:
 
 ```
-ensure => present
+ensure => "present"
```

Comment:
should we use double quotes here or singlequotes? (if that matters in this example). My impression is that single quotes are the best practice and puppet-lint also prefers them if there's no variable interpolation.

Reply:
Personally I differentiate between single and double and only use double when I need interpolation. But I think the general trend was to sacrifice the tiny amount of efficiency for ease of use. And I don't actually even know the real magnitude of the performance impact anyway.

In other words, I don't have a strong opinion.